app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -114,6 +117,8 @@ class SettingsFragment : PreferenceFragmentCompat() {
         showToolbar(getString(R.string.settings_title))
 
         update()
+
+        view!!.findViewById<RecyclerView>(R.id.recycler_view)?.hideInitialScrollBar(lifecycleScope)
```

Comment:
can we use the kotlin view getter here? `view?.recycler_view?.hideInitialScrollbar(lifecycleScope)`

Reply:
Unfortunately, no. I tried this, and it doesn't work. My guess is that it's because *we* don't have direct access to the recycler view. This is actually one Android is generating based on the build in PreferenceFragment.